pkg/compiler/global_test.go

```diff
@@ -219,6 +750,36 @@ func TestMultipleFiles(t *testing.T) {
 }
 
 func TestExportedVariable(t *testing.T) {
+	/*
+		t.Run("Use", func(t *testing.T) {
+			src := `package foo
+			var A = f()
+			var B int
+			func Main() int {
+				return B
+			}
+			func f() int {
+				B = 3
+				return B
+			}`
+			eval(t, src, big.NewInt(0))
+		})
+		t.Run("Use", func(t *testing.T) {
+			src := `package foo
+			import "github.com/nspcc-dev/neo-go/pkg/compiler/testdata/multi"
+			var A = 1
+			var B = f() + multi.SomeVar12 + multi.SomeVar30 + A
+			var C int
+			func Main() int {
+				return C
+			}
+			func f() int {
+				C = 3
+				return C
+			}`
+			eval(t, src, big.NewInt(0)) // TODO: fix this in a separate issue
```

Comment:
Issue number?

Reply:
These two tests are removed, they are added as an example to https://github.com/nspcc-dev/neo-go/issues/2661.

Reply:
We better keep them in some way (#2661 comment or code-level comment with a reference to #2661), we'll be fixing #2661 eventually and these tests will be helpful for this.

Reply:
See https://github.com/nspcc-dev/neo-go/issues/2661#issuecomment-1231527851.